### PR TITLE
faq.md: no need to pin version of zot registry docs

### PR DESCRIPTION
**Pull request by mikemccracken**
minor tweak: the zot registry url redirects to v1.4.3 right now, but as releases continue, this will become stale. Leaving it bare will send folks to the latest version of the docs.

### CODE REVIEW

**Comment by zr-msft** [comment]
Thank you @mikemccracken 

Change LGTM, but you need to sign-off on your commit before we can merge

**Comment by mikemccracken** [comment]
> Thank you @mikemccracken
> 
> Change LGTM, but you need to sign-off on your commit before we can merge

Hi, thanks. sorry about that, I had signed-off as my personal account but the author was set to my Cisco email. It should be OK now.

**Review by zr-msft — APPROVED** [review]

**Pull request merged**
